src/Tasks/BootstrapperUtil/BootstrapperBuilder.cs

```diff
@@ -442,15 +442,24 @@ public string[] GetOutputFolders(string[] productCodes, string culture, string f
             var files = new List<string>();
             BuildPackages(settings, null, null, files, null);
 
+            List<string> packagePaths = new List<string>() { invariantPath };
+            packagePaths.AddRange(Util.AdditionalPackagePaths.Select(p => Util.AddTrailingChar(p.ToLowerInvariant(), System.IO.Path.DirectorySeparatorChar)));
```

Comment:
What's the motivation for `ToLowerInvariant` here? It's a loc problem (Turkish `İ`), and the filesystem is already case-insensitive (usually, for NTFS and friends on Windows).

Reply:
Ah, I see you're following the existing pattern. Do you know if it's required for some reason? If it is, please comment why--and do the normalization when you read the reg key, rather than on every use.

Reply:
As you say, I'm doing it this way to follow the existing pattern.  I'm also not sure why ToLowerInvariant is necessary here.

Also, since the other code path that uses AdditionalPackagePaths (see RefreshProducts) doesn't call ToLowerInvariant on them I'm going to leave the call here.